docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -599,29 +601,44 @@ This command does not depend on the action of any of the KubeStellar controllers
 The following session demonstrates usage, including idempotency.
 
 ```shell
-kubectl ws root:imw-1
+KUBECONFIG=$imw_space_config kubectl kubestellar remove location demo1
 ```
 ``` { .bash .no-copy }
-Current workspace is "root:imw-1".
+synctarget.workload.kcp.io "demo1" deleted
+location.scheduling.kcp.io "demo1" deleted
 ```
 
 ```shell
-kubectl kubestellar remove location demo1
+KUBECONFIG=$imw_space_config kubectl kubestellar remove location demo1
 ```
-``` { .bash .no-copy }
-synctarget.workload.kcp.io "demo1" deleted
-location.scheduling.kcp.io "demo1" deleted
+
+## Getting a kubeconfig of a given space
+
+To access a particular space the kubeconfig file of that space needs to be
+obtained.  This can be done manually as follows: 
+
+```shell
+secret_name=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.name}")
+secret_namespace=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.namespace}")
+kubectl --kubeconfig $sm_core_config --context $sm_context get secret ${secret_name} -n ${secret_namespace} -o jsonpath='{$.data.kubeconfig}' | base64 -d | base64 -d > imw.kubeconfig
 ```
 
+Alternatively, the kubectl-kubestellar-get-config-for-space can be used. This script takes
+as input the name of the space (--space-name), the name of the space provider (--provider-name) 
```

Comment:
Why is the provider name even an option? I thought that KubeStellar insists that there is just one provider and its name is "default". If that is right then this command does not need a flag for the provider name.